extensions/amp-wordpress-embed/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ampproject/wg-bento'}],
```

Comment:
Is there a WP owner that would be appropriate to include?

Reply:
Could you please suggest me which name/names should I use here?

Reply:
Why not @ediamin? You could add me as well.

Reply:
Updated. Added westonruter as a owner.